pkg/utils/secrets/basic_auth.go

```diff
@@ -58,11 +59,17 @@ func (s *BasicAuthSecretConfig) Generate() (DataInterface, error) {
 		return nil, err
 	}
 
+	auth, err := utils.CreateBcryptCredentials([]byte(s.Username), []byte(password))
```

Comment:
I noticed that affected secrets are not regenerated automatically, when switching to Bcrypt. Is this intended?

Reply:
Yes, I planned to keep the secrets including their `auth` strings as they are until they are rotated for the next time.
Afaik, we did the same when we increased the RSA key size to 3072.